src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1971,6 +1970,9 @@ private bool InsertJsonFileFromPath(string fileContents, string filePath, bool f
                         {
                             ExtraWorkspaceViewInfo viewInfo = ExtraWorkspaceViewInfoFromJson(fileContents);
 
+                            // Update (assign new) GUIDs for each node, connection, note and group
```

Comment:
would you also add comments as to why for future reference?

Reply:
Updated, let me know if it is still vague or not clear.